src/ExtensionsSample/Models/ItemDoc.cs

```diff
@@ -10,6 +10,9 @@ public class ItemDoc
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        [JsonProperty("partition")]
+        public string Partition { get; set; }
```

Comment:
I don't see any samples using ItemDoc. DocumentDBSamples are using Item (same as easytables). Update docdb samples to use itemdoc